data/wicked/scripts/ifcfg/test-1.2/test.sh

```diff
@@ -0,0 +1,1166 @@
+#!/bin/bash
+
+
+test_description()
+{
+	cat - <<-EOT
+
+	Infiniband configuration parsing checks.
+	  * see: https://gitlab.suse.de/wicked-maintainers/wicked/-/wikis/infiniband-names
```

Comment:
Is there a publicly accessible link which could be used here to make the explanation useful for external contributors as well?

Reply:
Not now, it could be the [ipoib manpage](https://github.com/openSUSE/wicked/blob/master/man/src/ifcfg-ipoib.5.md#infinibandipoib-interfaces), but not so detailed with all these examples and possibilities. Maybe we should just public this list in github wiki as well :thinking: